packages/cloud_functions/cloud_functions/lib/src/firebase_functions.dart

```diff
@@ -96,6 +96,44 @@ class FirebaseFunctions extends FirebasePluginPlatform {
         delegate.httpsCallableWithUri(_origin, uri, options));
   }
 
+  /// A reference to the streaming Callable HTTPS trigger with the given name.
+  ///
+  /// Should be the name of the Callable function in Firebase that supports streaming.
+  HttpsCallableStream httpsStreamCallable(
+    String name, {
+    HttpsCallableOptions? options,
+  }) {
+    options ??= HttpsCallableOptions();
+    assert(name.isNotEmpty);
+    return HttpsCallableStream._(
+        delegate.httpsStreamCallable(_origin, name, options));
+  }
+
+  /// A reference to the streaming Callable HTTPS trigger with the given name.
+  ///
+  /// Should be URL of the 2nd gen Callable function in Firebase.
+  HttpsCallableStream httpsCallableStreamFromUrl(
+    String url, {
+    HttpsCallableOptions? options,
+  }) {
+    options ??= HttpsCallableOptions();
+    final uri = Uri.parse(url);
+    return HttpsCallableStream._(
+        delegate.httpsStreamCallableWithUri(_origin, uri, options));
+  }
+
+  /// A reference to the streaming Callable HTTPS trigger with the given name.
+  ///
+  /// Should be Uri of the 2nd gen Callable function in Firebase.
+  HttpsCallableStream httpsCallableStreamFromUri(
```

Comment:
Here we should copy the entire doc from the java implementation (and adapt it if needed)